### DISCUSSION TITLE
Show only file differences - ignore directory/folder differences

### DISCUSSION THREAD

**Discussion by Rock5** [WinMerge/winmerge#888] [Q&A]
I'm new to WinMerge so I might not be doing things right but WinMerge shows a lot, if not all, the folders are different even when the files are the same. Is it possible to only show the folders where there are files that are different? If there is an option for that, I can't see it.

I should note that I am showing only "Show Different Items" because there are files in one folder that don't exist in the other. I only want to see differences in the files that exist in both. On second thought, this may be why it's showing the folders are different. The question remains, though, is it possible to only show folders that have different files, given the filtering options I have selected?

Thanks.

**Comment by sdottaka** [comment]
If I understand your question correctly, unfortunately, WinMerge does not have the features you expect.
